roles/openshift_logging/defaults/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+image_prefix: docker.io/openshift/origin-
+image_version: latest
+logging_use_ops: False
+master_url: https://kubernetes.default.svc.cluster.local
+public_master_url: https://localhost:8443
```

Comment:
Does this need to be configurable? Seems like this might be better defaulted from something in openshift_facts rather than defaulting to localhost.